testsuite/tests/parsetree/source.ml

```diff
@@ -7372,3 +7372,7 @@ let f = function
 
 let () =
   f (fun (type t) -> x)
+
```

Comment:
I think it would be nice to add a comment with the issue and PR number to help future readers.